pkg/cmd/dev/builder.go

```diff
@@ -92,20 +90,10 @@ func (d *dev) getDockerRunArgs(
 	if err != nil {
 		return
 	}
-	buf, err := d.os.ReadFile(filepath.Join(workspace, "build/teamcity-bazel-support.sh"))
+	bazelImage, err := d.os.ReadFile(filepath.Join(workspace, "build", ".bazelbuilderversion"))
```

Comment:
Nit: Please call `strings.TrimSpace()` on this, just in case someone ends up introducing a newline or whatever while updating the file.